README.md

```diff
@@ -75,4 +75,41 @@ Before setting up the Titanium SDK, you must generate the appropriate credential
             visualLevel: onesignal.LOG_LEVEL_NONE
         });
     ```   
+1. Receive notifications callback:   
+(does not work on iOS when the app is closed (swiped away). But works fine when the app is running on background)   
+Opened:   
+    ```js
+        onesignal.addEventListener("OneSignalNotificationOpened",function(evt){
```

Comment:
@chmiiller I would like to suggest to change this event to 'notificationopened' without the prefix of 'onesignal' because the event is already set to the `onesignal` module. What do you think?